ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -57,15 +58,31 @@ function WorkloadCvesOverviewPage() {
             <PageTitle title="Workload CVEs Overview" />
             {/* Default filters are disabled until fixability filters are fixed */}
             <Divider component="div" />
-            <PageSection variant="light" padding={{ default: 'noPadding' }}>
-                <Flex direction={{ default: 'column' }} className="pf-u-py-lg pf-u-pl-lg">
-                    <FlexItem>
-                        <Title headingLevel="h1">Workload CVEs</Title>
-                    </FlexItem>
+            <PageSection
+                className="pf-u-display-flex pf-u-flex-direction-row pf-u-align-items-center"
+                variant="light"
+                padding={{ default: 'noPadding' }}
+            >
+                <Flex
+                    direction={{ default: 'column' }}
+                    className="pf-u-py-lg pf-u-pl-lg pf-u-flex-grow-1"
+                >
+                    <Title headingLevel="h1">Workload CVEs</Title>
                     <FlexItem>
                         Prioritize and manage scanned CVEs across images and deployments
                     </FlexItem>
                 </Flex>
+                <FlexItem className="pf-u-pr-lg">
+                    <Button
+                        variant="secondary"
+                        onClick={() => {
+                            setDefaultWatchedImageName('');
+                            watchedImagesModalToggle.openSelect();
```

Comment:
A question for understanding. This borrows and applies hook for Select to actions and state for modal?

Reply:
Yes, this is the unfortunately named `useSelectToggle` that is not PatternFly or Select specific. It really should be called `useToggle()` as it is a general purpose toggle hook.

This has been on my refactor list and every time I use it again I realize that I should take care of the renaming.